sql/core/src/test/scala/org/apache/spark/sql/internal/ExecutorSideSQLConfSuite.scala

```diff
@@ -191,6 +191,52 @@ class ExecutorSideSQLConfSuite extends SparkFunSuite with SQLTestUtils {
       assert(checks2.forall(_.toSeq == Seq(true, true)))
     }
   }
+
+  test("SPARK-44897 propagate local properties to subquery broadcast execuction thread") {
```

Comment:
if this test can't reproduce the bug directly, we can put it in the PR descriptions with instructions about the extra change to reproduce the bug (add sleep).

Reply:
Ok. Does it make sense to leave this test in the code still or do you think putting it in the description with instructions is enough

Reply:
Please remove the test case from the code, @ChenMichael . PR description (with your reproducible instruction is enough).

Reply:
👍 Done